extensions/mssql/src/objectManagement/ui/objectManagementDialogBase.ts

```diff
@@ -20,6 +20,7 @@ function getDialogName(type: ObjectManagement.NodeType, isNewObject: boolean): s
 }
 
 export interface ObjectManagementDialogOptions extends ScriptableDialogOptions {
+	command?: string;
```

Comment:
What does "command" mean in this context? Is this supposed to be the dialog type that I suggested adding? If so, would suggest making a const string enum instead of `DialogType` with the values `new`, `edit` and `properties`.

And ideally this should be required as well - it'll be a lot better if we can rely on there always being a type of some sort.

Reply:
Yes, this is to identify the dialog type that we are trying to open, and helpful to handle the respective dialog.ts and also will be used in the STS to call the respective dialog handler. I'm changing it to string Enum as suggested, Thanks :)

Reply:
"dialog options" doesn't sound like something that should go up to STS directly. Where are these options being passed up?

Maybe `action` is a better property name if it's going to be passed along then. (command has a very specific meaning in ADS so we should avoid using that unless we mean actual ADS commands)

Reply:
We use these options to intializeView() which will be passed to STS, there I use this option.command (I have changed the name to dialogType(coming from Enum of Edit/New/Properites)) to differentiate databaseHandler and databasePropertiesHandler.

Reply:
Using isNewObject and existed Database Dialog resolved this one and no need of having new property to help in differentiate the database with its properties.